raven/utilities/forecasting.py

```diff
@@ -173,3 +182,79 @@ def perform_climatology_esp(model_name, forecast_date, forecast_duration, **kwds
     qsims["member"] = (["member"], avail_years)
 
     return qsims
+
+def get_hindcast_day(region,date, climate_model='GEPS'):
+    '''
+    This function generates a forecast dataset that can be used to run raven. 
+    Data comes from the CASPAR archive and must be aggregated such that each file
+    contains forecast data for a single day, but for all forecast timesteps and
+    all members.
+    
+    The code takes the region shapefile, the forecast date required, and the 
+    climate_model to use, here GEPS by default, but eventually could be 
+    GEPS, GDPS, REPS or RDPS.
+    '''
+
+    # Get the file locations and filenames as a function of the climate model and date
+    if climate_model is 'GEPS':
+        filein='https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/dodsC/birdhouse/caspar/daily/GEPS_' + dt.datetime.strftime(date,'%Y%m%d') + '.nc'
+        
+        # Here we also extract the times at 6-hour intervals as Raven must have 
+        #constant timesteps.
+        times=[]
+        for single_time in (date + dt.timedelta(hours=n) for n in range(0,384,6)):
+            times.append(single_time)
+            
+    elif climate_model is 'GDPS':
+        filein='https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/dodsC/birdhouse/caspar/daily/GDPS_' + dt.datetime.strftime(date,'%Y%m%d') + '.nc'
+        
+    elif climate_model is 'RDPS':
+        filein='https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/dodsC/birdhouse/caspar/daily/REPS_' + dt.datetime.strftime(date,'%Y%m%d') + '.nc'
+        
+    elif climate_model is 'REPS':
+        filein='https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/dodsC/birdhouse/caspar/daily/RDPS_' + dt.datetime.strftime(date,'%Y%m%d') + '.nc'
```

Comment:
replace by 
```python
filein = f"https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/dodsC/birdhouse/caspar/daily/{climate_model}_{date:%Y%m%d}.nc"
```

Reply:
Are we waiting for a "latest view" to be created here?

Reply:
I would not wait for it, but we'll update the link once it is up there.